gulpfile.js

```diff
@@ -166,6 +166,14 @@ gulp.task('compile:static', () =>
 gulp.task('compile', gulp.series('compile:ts', 'compile:static', 'compile:dynamic'));
 
 async function runWebpack(packages) {
+
+  // add the entrypoints common to both vscode and vs here
+  packages.push(...[
```

Comment:
I'd suggest this to avoid mutating the input, just in case anyone tries to use it later

```suggestion
  packages = [
      ...packages,
```